cmds/cpu/cpu.go

```diff
@@ -439,6 +439,11 @@ func newCPU(host string, args ...string) error {
 	if len(*cpudCmd) > 0 {
 		c.WithCpudCommand(*cpudCmd)
 	}
+	if len(*fstab) > 0 {
```

Comment:
honestly, the if is not really needed, but well ... no harm.

Reply:
can see either way.  you've got the len check elsewhere already